tasks/system_probe.py

```diff
@@ -348,11 +348,17 @@ def build_object_files(ctx, bundle_ebpf=False):
         "offset-guess",
     ]
     bindata_files = [
+        os.path.join(bpf_dir, "tcp-queue-length-kern-user.h"),
```

Comment:
This is already in the list, two lines down.

Reply:
🤦